docs/contributing.rst

```diff
@@ -326,3 +326,73 @@ escaping and without iterated replacements. An area can be delimited by ``<#name
 by as many concatenations of its contents as there were sets of variables supplied to the template system,
 each time replacing any ``<inner>`` items by their respective value. Top-level variables can also be used
 inside such areas.
+
+.. _documentation-style:
+
+Documentation Style Guide
+=========================
+
+The following are style recommendations specifically for documentation
+contributions to Solidity.
+
+British English
+---------------
+
+Use British English, unless using project or brand names. For a good guide to
+what this means read `this guide from Oxford dictionaries <https://en.oxforddictionaries.com/spelling/british-and-spelling>`_
```

Comment:
Do we really need to mention what British English is? Perhaps we could also add something like "If possible, use word forms that are valid in both American and British English."

Reply:
I believe we do, yes, and I'm not sure the suggestion really makes sense, made a slight change to clarify and make it less prescriptive.